google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITWriteTest.java

```diff
@@ -320,6 +323,23 @@ public void writeStringNull() {
     assertThat(row.isNull(0)).isTrue();
   }
 
+  @Test
+  public void writeJson() {
+    assumeFalse("Emulator does not yet support JSON", EmulatorSpannerHelper.isUsingEmulator());
+    write(baseInsert().set("JsonValue").to(Value.json("{}")).build());
+    Struct row = readLastRow("JsonValue");
+    assertThat(row.isNull(0)).isFalse();
+    assertThat(row.getJson(0)).isEqualTo("{}");
+  }
+
+  @Test
+  public void writeJsonNull() {
+    assumeFalse("Emulator does not yet support JSON", EmulatorSpannerHelper.isUsingEmulator());
+    write(baseInsert().set("JsonValue").to(Value.json(null)).build());
+    Struct row = readLastRow("JsonValue");
+    assertThat(row.isNull(0)).isTrue();
```

Comment:
nit: Also add an assertion that the type of the column is recognized as JSON. Considering the fact that this type partly overlaps with `String`, I think it's good to have an assertion that verifies that we can never accidentally make any changes that would make the distinction between a null `String` and null `JSON` fail.